frontend/src/components/node/List.tsx

```diff
@@ -26,6 +26,7 @@ export default function NodeList() {
         {
           id: 'cpu',
           label: t('CPU'),
+          gridTemplate: 'minmax(120px, .5fr)',
```

Comment:
Why is this needed?

Reply:
Otherwise we get this in the UI (described in the issue):

![image](https://github.com/user-attachments/assets/5de4696d-30e1-4be7-b182-d1adef69bcb9)

Reply:
Maybe try `gridTremplate: 'min-content'`?

Reply:
seems to look very similar, made the change here for simplicity